src/NServiceBus.Core/Pipeline/PipelineSettings.cs

```diff
@@ -25,6 +25,7 @@ public void Remove(string stepId)
         {
             // I can only remove a behavior that is registered and other behaviors do not depend on, eg InsertBefore/After
             Guard.AgainstNullAndEmpty(nameof(stepId), stepId);
+            EnsureWriteEnabled(stepId, "remove");
```

Comment:
think you can use this instead:

`EnsureWriteEnabled(stepId, nameof(Remove)); //the nameof(Remove) would be "Remove"`

Same goes for all the below methods.

Reply:
Yeah, you are right, changed